package.json

```diff
@@ -4,7 +4,7 @@
   "description": "An express-based Node.js API bootstrapping module for microservices.",
   "main": "index.js",
   "scripts": {
-    "test": "NODE_ENV=test ./node_modules/.bin/mocha --timeout 2000 --colors --recursive -R spec --ui bdd 'test/**/*.test.js'",
+    "test": "NODE_ENV=test ./node_modules/.bin/mocha --timeout 2000 --colors --recursive -R spec --ui bdd 'test/**/*.test.js' --exit",
```

Comment:
Would be nice to move these mocha opts into a mocha.opts file within the test directory, then any runner of the tests can use the options like IDE test runners?